aggregator/handler/config.go

```diff
@@ -171,6 +175,41 @@ func (c flushHandlerConfiguration) Validate() error {
 	return nil
 }
 
+type trafficControlConfiguration struct {
+	DefaultDisabled   bool                     `yaml:"defaultDisabled" validate:"nonzero"`
```

Comment:
Hm validate nonzero here means this can't be set to false in the config, that's probably not what you want? It's probably better to use a `*bool` and check for nil explicitly.